profiles/Tier30/T30_Warrior_Arms.simc

```diff
@@ -108,24 +108,27 @@ actions.hac+=/wrecking_throw
 actions.single_target=sweeping_strikes,if=spell_targets.whirlwind>1
 actions.single_target+=/mortal_strike
 actions.single_target+=/rend,if=remains<=gcd|talent.tide_of_blood&cooldown.skullsplitter.remains<=gcd&(cooldown.colossus_smash.remains<=gcd|debuff.colossus_smash.up)&dot.rend.remains<dot.rend.duration*0.85
-actions.single_target+=/avatar,if=talent.warlords_torment&rage.pct<33&(cooldown.colossus_smash.ready|debuff.colossus_smash.up|buff.test_of_might.up)|!talent.warlords_torment&(cooldown.colossus_smash.ready|debuff.colossus_smash.up)
+actions.single_target+=/avatar,if=cooldown.colossus_smash.ready|debuff.colossus_smash.up|buff.test_of_might.up
```

Comment:
Avatar was often down during Arms windows where it's most beneficial. This change increased the benefit from ~20% to ~26%.